packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.test.js

```diff
@@ -19,8 +19,10 @@ describe('calculateDomain', () => {
   it('should be computed from data and series option', () => {
     const calculatedDomains = domains(
       [argumentAxis, valueAxis],
-      [{ axisName: 'valueAxis', argumentField: 'arg', valueField: 'val' }],
-      [{ arg: 1, val: 9 }],
+      [{
+        axisName: 'valueAxis', argumentField: 'arg', valueField: 'val', name: 'name',
+      }],
+      [{ arg: 1, val: 9, valnameend: 9 }],
```

Comment:
A valnameend is not a readable string. I think "val-name-end" string is the more clarifying string